### DISCUSSION TITLE
Vaex Example not working on Kaggle/Colab

### DISCUSSION THREAD

**Discussion by dharik-arsath** [vaexio/vaex#1482] [Q&A]
import vaex

data = vaex.example() # is not working on kaggle and colab and the error it shows is "Cannot open /root/.vaex/data/helmi-dezeeuw-2000-FeH-v2-10percent.hdf5 nobody knows how to read it."  It downloads successfully from the github and failed to open. Please look into the issue, Thanks....

Note: it works on local machine and version is same in both local, kaggle and colab: " vaex-core 4.3.0 post-1"

**Comment by JovanVeljanoski** [comment]
I can't reproduce it. It works for me both in kaggle and colab..
Did you install only vaex-core? Just do `!pip install vaex`.

Also sometimes you might need to restart the runtime...

I just tried this minutes ago

<img width="1216" alt="image" src="https://user-images.githubusercontent.com/18574951/127826633-207226a8-d471-46cb-bfab-21df36cf107d.png">

and this is kaggle:
<img width="1111" alt="image" src="https://user-images.githubusercontent.com/18574951/127826704-ff73d803-2062-4b2b-bd93-f4217e7bd5bd.png">
